### PR TITLE
Feature/map view (#14)

### PR DESCRIPTION
**Pull request by jsjqian**
Adds the pretty print map for the view

Currently, it is just a scrollview with a png, not sure if we want to do something else with it, I am open to making any changes suggested to see if it would look better. We should end up locking the app to only be in portrait orientation so the map doesn't look funky when a user turns it sideways. Any criticism is welcome!

### CODE REVIEW

File: android/settings.gradle [hunk 1]
```diff
@@ -1,5 +1,8 @@
 rootProject.name = 'bitcampapp'
-
-include ':app'
 include ':react-native-vector-icons'
 project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
+include ':react-native-svg'
+project(':react-native-svg').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-svg/android')
+include ':app'
+include ':PDFView'
+project(':PDFView').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-pdf-view/android')
```
Comment (jdai8) [inline android/settings.gradle]: Are we using these libraries (react-native-svg, pdfview)? If not, we can take out a lot of this stuff.

File: scenes/MapScene.js [hunk 2]
```diff
@@ -1,18 +1,24 @@
 import React, { Component } from 'react';
-import { View, Text } from 'react-native';
+import { ScrollView, Text, Image, Dimensions } from 'react-native';
 
 export default class MapScene extends Component {
```
Comment (jdai8) [inline scenes/MapScene.js]: I think we can reduce this to just a function:
https://facebook.github.io/react/docs/components-and-props.html#functional-and-class-components

**Review by jdai8 — COMMENTED** [review]
I don't think we want to version control the index.android.bundle file.

We might use the svg or pdf libraries in the future, but if we plan on merging this the way it is now (using the png), I'd prefer if we take out the stuff related to those libraries.

I don't think we should a force a portrait mode. I feel like it's more natural to view the map horizontally, so the numbers and everything are right-side-up.

File: android/settings.gradle (hunk 1 above)
Comment (goccert25) [reply android/settings.gradle]: agree with ^

**Comment by jsjqian** [comment]
How do these changes look?

**Pull request merged**